my_project/tab_natural_ventilation/app_natural_ventilation.py

```diff
@@ -85,7 +86,7 @@ def inputs_tab():
                         className="mb-2",
                         n_clicks=1,
                     ),
-                    html.H6("Outdoor dry-bulb air temperature range"),
+                    html.H6("Outdoor dry-bulb air temperature range -- (default: °C) / °F"),
```

Comment:
this should not be hardcoded but it should be updated as a function of the user selection

Reply:
This problem is now solved. The set number will be changed automatically according to the si-ip-state.